docs/index.md

```diff
@@ -31,21 +31,21 @@ Mocha is a feature-rich JavaScript test framework running on [Node.js](https://n
 - [reports test durations](#test-duration)
 - [highlights slow tests](#dot-matrix)
 - [file watcher support](#min)
-- [global variable leak detection](#--check-leaks)
-- [optionally run tests that match a regexp](#-g---grep-pattern)
-- [auto-exit to prevent "hanging" with an active loop](#--exit----no-exit)
+- [global variable leak detection](#-check-leaks)
```

Comment:
Were all of these references wrong?

Reply:
The double-dash removal was introduced by configuring autogenerated heading slugs to use `slugify` across the two markdown tools. The other bigger changes are actually bug fixes. The heading changed, but this section is manually maintained, so the references didn't get updated

Reply:
Not a fan of the single dash ref conversion for cmdline args.
Not a fan of `slugify` either...
* `#-g---grep-pattern` => `#-grep-regexp-g-regexp` (Huh?)